### PR TITLE
(dev-doc-detector-threshold-configs): Updated issue platform detector docs.

### PR DESCRIPTION
**Pull request by Abdkhan14**
Updated issue platform detector docs to highlight detector threshold configurations.

### CODE REVIEW

File: src/docs/issue-platform-detectors.mdx [hunk 1]
```diff
@@ -127,3 +127,26 @@ default={
...
+
+![Detector Threshold settings customer ui screenshot](img/issuePlatformCustomerDetectorSettings.png)
+
+Also, add the corresponding issue to the admin settings ui. It allows support to enable/disable detection of the issue for a project:
+
+![Detector Threshold settings admin ui screenshot](img/issuePlatformAdminDetectionSettings.png)
+
+Things to note when making detector thresholds configurable: 
+
+- The **maximum threshold value** of the range, should not be big enough to make the issue undetectable. We are not trying to disable detection here.
```
Comment (vivianyentran) [inline src/docs/issue-platform-detectors.mdx:143]:
```suggestion
- The **maximum threshold value** of the range should not be big enough to make the issue undetectable. We are not trying to disable detection here. 
```

File: src/docs/issue-platform-detectors.mdx [hunk 2]
```diff
@@ -127,3 +127,26 @@ default={
...
+![Detector Threshold settings customer ui screenshot](img/issuePlatformCustomerDetectorSettings.png)
+
+Also, add the corresponding issue to the admin settings ui. It allows support to enable/disable detection of the issue for a project:
+
+![Detector Threshold settings admin ui screenshot](img/issuePlatformAdminDetectionSettings.png)
+
+Things to note when making detector thresholds configurable: 
+
+- The **maximum threshold value** of the range, should not be big enough to make the issue undetectable. We are not trying to disable detection here. 
+- The **minimum threshold value** of the range, should hold some significance when exceeded. We want our users to find a issue that we detect, significant enough to look further into.
```
Comment (vivianyentran) [inline src/docs/issue-platform-detectors.mdx:144]:
```suggestion
- The **minimum threshold value** of the range should hold some significance when exceeded. We want our users to find an issue that we detect significant enough to look further into. 
```

File: src/docs/issue-platform-detectors.mdx [hunk 3]
```diff
@@ -127,3 +127,26 @@ default={
...
+Also, add the corresponding issue to the admin settings ui. It allows support to enable/disable detection of the issue for a project:
+
+![Detector Threshold settings admin ui screenshot](img/issuePlatformAdminDetectionSettings.png)
+
+Things to note when making detector thresholds configurable: 
+
+- The **maximum threshold value** of the range, should not be big enough to make the issue undetectable. We are not trying to disable detection here. 
+- The **minimum threshold value** of the range, should hold some significance when exceeded. We want our users to find a issue that we detect, significant enough to look further into. 
+- The **impact** of changing the thresholds chosen to be made configurable, **should be easy for the user to interpret**. For example: Instead of displaying the raw duration of db base spans on the main thread
+ as a configurable threshold, we display the frame rate drop that may result from having a db span of that duration on the main thread:
```
Comment (vivianyentran) [inline src/docs/issue-platform-detectors.mdx:146]:
```suggestion
- The **impact** of changing the configurable thresholds **should be easy for the user to interpret**. For example, instead of configuring the raw duration of DB base spans on the main thread, users can configure the resulting frame rate drop. 
```

**Review by vivianyentran — APPROVED** [review]
Couple of small edits. Otherwise LGTM!

File: package.json [hunk 4]
```diff
@@ -1,11 +1,8 @@
 {
-  "name": "sentry-docs-develop",
```
Comment (udameli) [inline package.json:2]: is this change intentional?

Comment (Abdkhan14) [reply package.json]: Yeah this wasn't intentional. I ran yarn init, instead of yarn run init. Reverted this change.
